Gemfile

```diff
@@ -1,66 +1,66 @@
+# frozen_string_literal: true
+
 source 'https://rubygems.org'
 
-ruby '2.1.10'
+ruby '2.4.1'
 
-# Use grape for API
+gem 'dotenv-rails'
 gem 'grape', '0.17.0'
+gem 'rails', '~> 5.2.1'
 
-group :development do
-  gem 'letter_opener'
-end
-
-group :development, :test do
-  gem 'factory_girl_rails'
-  gem 'faker'
-  gem 'pry-nav'
-  gem 'pry-rails'
-  gem 'rubocop', '~> 0.46.0'
-end
-
-# Use rspec+selenium_webdriver+capybara as for test
-group :test do
-  gem 'capybara', '2.1.0'
-  gem 'codeclimate-test-reporter', '~> 1.0.0'
-  gem 'codecov', require: false
-  gem 'database_cleaner'
-  gem 'rspec-rails'
-  gem 'selenium-webdriver', '~> 2.37'
-  gem 'shoulda-matchers'
-  gem 'simplecov', require: false
-  gem 'sqlite3'
-end
+gem 'rubocop', '~> 0.50.0', require: false
+gem 'rubocop-rspec', require: false
 
-gem 'rails', '~> 4.2'
-gem 'rake', '< 11.0'
+# Reduces boot times through caching; required in config/boot.rb
+gem 'bootsnap', '>= 1.1.0', require: false
```

Comment:
@sudakov-eugene is there really any need of it in a small app like this? :laughing: 
I wonder if results cost having another additional dependency.

Reply:
Yeah, it's default in rails 5.2 and it really helps to get maximum performance improvement. You can check how it works here https://github.com/Shopify/bootsnap#how-does-this-work

Reply:
As I know, it doesn't clear cache after itself yet, but OK.
https://github.com/Shopify/bootsnap/issues/185